src/lib.rs

```diff
@@ -113,12 +143,22 @@
 //! # #![allow(unstable)]
 //! use semver::Version;
 //! use semver::VersionReq;
+//! use std::error::Error;
+//!
+//! fn try_compare() -> Result<(), Box<Error>> {
+//!
```

Comment:
This is a bit of a bit, but once we hide the method signatures in this example we'll need to remove this newline, otherwise the docs will have an empty line in them too.